system-test/storage.ts

```diff
@@ -2791,6 +2792,21 @@ describe('storage', () => {
     });
   });
 
+  describe('bucket upload with progress', () => {
+    it('show bytes sent', async () => {
```

Comment:
This test might need to be reworked, because it looks like it requires a human to watch the output and verify there is output? I think you could just check that the `bytesRead` argument in the `onUploadProgress` event is `> 0 && <= fileSize`.

Reply:
Reworked!